tests/test_service_v2.py

```diff
@@ -586,6 +586,15 @@ def test_update_entity(service):
     assert isinstance(request, pyodata.v2.service.EntityModifyRequest)
 
     request.set(Value=34)
+    # Tests if update entity is correctly call to_json method
+    request.set(Date=datetime.datetime(2017, 12, 24, 19, 0))
+
+    assert request._values['Value'] == 34
+    assert request._values['Date'] == '/Date(1514142000000)/'
+
+    # Tests if if passed perforated datetime, to json does not update given value (for backward compatibility reasons)
```

Comment:
Would be nice to rephrase this comment - e.g. explanation of what is `perforated datetime` would be good, same for `to json`, is it name of the method?